sarif/sarif_file_utils.py

```diff
@@ -23,7 +23,9 @@ def combine_code_and_description(code: str, description: str) -> str:
         code = code.strip()
         length_budget -= len(code) + 1
     # Allow extra space when truncating for continuation characters
-    length_budget_pre_continuation = length_budget - 20
+    length_budget_pre_continuation = length_budget - 5
```

Comment:
What is the magic number 5? Maybe `" ..."` should be a constant and `5` should be calculated based on its length?